### PR TITLE
moved and renamed caching service template

### PR DESCRIPTION
**Pull request by rhoads-zach**
@lance @lholmquist Let me know what you think

### CODE REVIEW

**Comment by Ladicek** [comment]
@rhoads-zach This is wrong, the `.openshiftio/service.cache.yaml` file needs to stay -- without it, Launcher wouldn't launch the JDG service and the booster wouldn't work. We need a _copy_ of that file in the root directory.

**Comment by rhoads-zach** [comment]
@Ladicek amended the PR.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ## Cache Mission
 
 The greeting-service requires a running JDG server. In OpenShift, you
-can create one with `oc create -f greeting-service/.openshift/service.jdg.yaml`.
```
Comment (lance) [inline README.md]: The `.openshift/service.jdg.yaml` file is expected to be in that directory by `nodeshift` I think. @lholmquist correct me if I am wrong in this case, but as a rule that's true. Not sure how easy that will be to change at the moment, but we need to look at cost vs. benefit :)

Comment (Ladicek) [reply README.md]: There's no `.openshift` anyway :-) The `.openshiftio` is needed for Launcher, and that is still present. Also Launcher strips all the `.openshiftio` directories when creating the .zip / Git repo, so this PR adds a copy to the root dir.

Comment (lance) [reply README.md]: I just scanned the nodeshift code, and I could be completely wrong about it having some kind of dependency on `.openshift`. Foggy brain!

Comment (lholmquist) [reply README.md]: like @Ladicek said,  the reason for the .openshiftio directory is for the launcher.  man, that is confusing

**Comment by lholmquist** [comment]
@rhoads-zach 👍

**Pull request merged**